sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -177,6 +179,7 @@ public boolean deactivateUserByLogin(DbSession dbSession, String login) {
     mapper.deleteUserMeasureFilterFavourites(dto.getId());
     mapper.deleteUserProperties(dto.getId());
     mapper.deleteUserRoles(dto.getId());
+    mapper.deletePropertiesMatchingLogin(asList(DEFAULT_ISSUE_ASSIGNEE), dto.getLogin());
```

Comment:
why not use the properties mapper? We're in a hybrid case, where there should be a service to query several tables.
Normally, one mapper -> one table (Purge is a notable exception)